app/assets/javascripts/periodicals.js

```diff
@@ -119,6 +119,11 @@ $(document).ready(function() {
                         $(".heap-used", thisHeap).text(numeral(data.used).format("0,0"));
                         $(".heap-total", thisHeap).text(numeral(data.total).format("0,0"));
                         $(".heap-max", thisHeap).text(numeral(data.max).format("0,0"));
+
+                        var journal = $(".journal-uncommitted", thisHeap);
+                        if (journal.length > 0) {
+                            journal.text(numeral(data.journal_uncommitted_entries).format("0,0"));
+                        }
```

Comment:
I find confusing mixing heap and journal information under a `updateNodeHeapUsage` method, could you please change that? :)